src/v3/src/components/OpenOktaVerifyFPButton/OpenOktaVerifyFPButton.tsx

```diff
@@ -40,18 +40,19 @@ const OpenOktaVerifyFPButton: UISchemaElementComponent<{
       step,
       href,
       challengeMethod,
+      label,
     },
   } = uischema;
   const [key, setKey] = useState<number>(0);
-  const label = getTranslation(translations, 'label');
+  const btnLabel = label || getTranslation(translations, 'label');
```

Comment:
Why can't this using the `translations` transformer to set the label text?

Reply:
refactored it to use the translation transformer instead